app/controllers/errors_controller.rb

```diff
@@ -0,0 +1,6 @@
+class ErrorsController < ApplicationController
+  def show
+    @exception = env['action_dispatch.exception']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.